tests/ragger/requirements.txt

```diff
@@ -1,4 +1,4 @@
-ragger[speculos]>=1.7.0,<1.8.0
+ragger[speculos]>=1.8.0,<1.9.0
```

Comment:
Why not simply latest ?

Reply:
To prevent an update with breaking changes on the implemented tests, but if this is unrealistic I could indeed change it. :+1:

Reply:
I think we can use latest, we will never introduce breaking changes, if we really need to we'll start again from `2.0.0` anyway and it won't be a sudden update